libr/asm/arch/6502/6502dis.c

```diff
@@ -1,125 +1,163 @@
+/* radare - LGPL - Copyright 2015 - condret, Ricardo Quesada, qnix */
```

Comment:
I don't think I have contributed to that file... feel free to remove me. thanks.